backend/Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN mkdir /build
 ADD . /build
 WORKDIR /build/cmd
 RUN apk add gcc musl-dev
-RUN go test
+RUN go test ../...
```

Comment:
Is `../...` necessary? Wouldn't `./...` be enough? You don't have .go files outside of `backend` directory